hack/generate-changelog.sh

```diff
@@ -64,11 +57,17 @@ while [[ $# -gt 0 ]]; do
 done
 
 # Fetch the latest tags from the remote
-git fetch $REMOTE --tags
+remote=$( git remote -v | grep -E "NVIDIA/k8s-device-plugin(\.git)?\s" | grep -oE "^[a-z]+" | sort -u )
```

Comment:
```bash
❯ git remote -v | grep -E "NVIDIA/k8s-device-plugin(\.git)?\s" | grep -oE "^[a-z]+" | sort -u
upstream
```

Reply:
Yes, this is intended to get the remote to use for fetching the tags. This should address us not having `origin` as our default remote.